README.md

```diff
@@ -436,56 +437,53 @@ let test#project_root = "/path/to/your/project"
 
 ### Language-specific
 
+Test.vim does not automatically detect runners. User must explicitly define desired runner. By default the first available runner will be chosen where no runners are specified.
```

Comment:
Could you give some examples where it does not automatically detect runners? One of the key features of this plug-in as stated in the README is:

> Wide range of test runners which are automagically detected

If there are deficiencies in the detection I'd rather address